ee/cis/macos-13/cis-policy-queries.yml

```diff
@@ -2007,3 +2007,34 @@ spec:
   purpose: Informational
   tags: compliance, CIS, CIS_Level1, CIS6.3.6
   contributors: artemist-work
+---
+apiVersion: v1
+kind: policy
+spec:
+  name: CIS - Ensure Show Full Website Address in Safari Is Enabled (MDM, FDA Required)
```

Comment:
FDA is only required for the terminal method, since we are using profile and managed_policies then there's no need for it AFAICS:
```
Note: To run the Terminal commands, Terminal must be granted Full Disk Access in the Security & Privacy pane in System Preferences.
```

Reply:
Of course.
removing...